solo/cli/__init__.py

```diff
@@ -20,19 +20,9 @@
 from solo.cli.program import program
 
 from . import _patches  # noqa  (since otherwise "unused")
+from ._warnings import init_checks
```

Comment:
from `_checks`, right?

Reply:
I was wondering what will be a better name `_checks` or `_warnings`.
We are performing checks, but the side effect is warnings, so let me know what you prefer.